siliconcompiler/core.py

```diff
@@ -473,7 +487,7 @@ def create_cmdline(self, progname, description=None, switchlist=None, input_map=
                     self.set(*args, val, clobber=True)
 
     #########################################################################
-    def find_function(self, modulename, funcname, moduletype=None):
+    def find_function(self, modulename, funcname, moduletype=None, modulestep=None):
```

Comment:
is `modulestep` used?

Reply:
Yes, to define the task name. Now that we are only running the task setups, this probably shouldn't be optional, and it should be moved in front of `moduletype`.

Reply:
should it be named `moduletask` instead then?